### PR TITLE
Export Polygon

### PR DESCRIPTION
**Pull request by lukahartwig**
This PR adds a feature for exporting polygons in multiple formats (LatLng, GeoJSON and JTS).

### CODE REVIEW

**Comment by lukahartwig** [comment]
Got some Design feedback from Rafael. Will implement some UI feedback on the "copy to clipboard" action.

File: src/common/components/ChevronDownIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    className?: string;
+}
+
+export const ChevronDownIcon: React.FC<Props> = props => {
+    return (
+        <svg color={AUTHENTIC_BLUE_900} width={24} height={24} viewBox="0 0 24 24" {...props}>
+            <path d="M12 13.726l7-6.175v2.676l-7 6.222-7-6.222V7.55z" fill="currentColor" fillRule="nonzero" />
+        </svg>
+    );
+};
```
Comment (medhoover) [inline src/common/components/ChevronDownIcon.tsx]: For consistency, we could use the same syntax as in: src/ActionBar/Icons/Export.tsx

File: src/conversion/format.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+export interface Format {
+    name: FormatType;
+    displayName: string;
+    description?: string;
+    serialize: (coordinates: Coordinate[]) => string;
+    deserialize: (raw: string) => Coordinate[];
+    validate: (value: string) => boolean;
+}
+
+export const format: Record<FormatType, Format> = {
```
Comment (medhoover) [inline src/conversion/format.ts]: Having the serializer and the deserializer in a one format reference is a smart idea 👍.

File: src/common/components/ChevronDownIcon.tsx (hunk 1 above)
Comment (lukahartwig) [reply src/common/components/ChevronDownIcon.tsx]: The icons in the `ActionBar` all have to have a certain interface due to how the `ActionBar` works. This icon is copied from the internal design system, so we could potentially replace this icon in the future.

**Pull request merged**